package-lock.json

```diff
@@ -2271,9 +2271,9 @@
       "dev": true
     },
     "@types/node": {
-      "version": "8.10.45",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-8.10.45.tgz",
-      "integrity": "sha512-tGVTbA+i3qfXsLbq9rEq/hezaHY55QxQLeXQL2ejNgFAxxrgu8eMmYIOsRcl7hN1uTLVsKOOYacV/rcJM3sfgQ==",
+      "version": "8.10.46",
```

Comment:
you must not push package-lock.json file unless you modify package.json.

Reply:
Package lock file has been reverted.